spec/Records.tex

```diff
@@ -57,45 +56,22 @@ \section{Record Declarations}
 \end{syntax}
 
 A \sntx{record-declaration-statement} defines a new type symbol specified by the
-identifier.  A record inherits data and methods from other records
-if the \sntx{record-inherit-list} is specified.
-
-\begin{future}
-Allowing a record to inherit from more than one record is future work.
-\end{future}
-
-\begin{rationale}
-We do not allow records to inherit from classes because of the following.
-
-Inheritance implies that the derived type can be cast to one of its base types.
-If the base type is a record type, casting to the base type has the effect of
-removing all of the data fields and all of functions that are not associated
-with the base type.  Thereafter, the record variable has the base record type,
-in both compile-time and run-time interpretations.
-
-If the base type were a class type, the result of the cast would have the static
-type of the base class while its run-time type was a record type.  Since a
-record's type is supposed to be determined at compile time, this is a bit
-incongruous with the definition of a record.  Moreover, space would have to be
-allocated in this special case, to store the record's run-time type.
-\end{rationale}
+identifier.
 
 As in a class declaration, the body of a record declaration can contain
 variable, iterator and method declarations as well as nested type declarations.
```

Comment:
Let us simplify to "... can contain variable, method and type declarations."

and merge this sentence's paragraph with the previous paragraph.